components/base/sensorcontrolled/spin.go

```diff
@@ -34,6 +34,17 @@ func (sb *sensorBase) Spin(ctx context.Context, angleDeg, degsPerSec float64, ex
 	}
 
 	sb.setPolling(true)
+
+	// starts a goroutine from within wheeled base's runAll function to run motors in the background
+	if err := sb.startRunningMotors(ctx, angleDeg, degsPerSec); err != nil {
+		return err
+	}
+
+	if angleDeg > 0 {
```

Comment:
Add a comment above that this will be removed when we migrate spin to use the control loop